src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+/* reset browser default margins */
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
- Don't use * selector (it impacts performance)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/02de8e44-3fd6-43fd-af53-1046b4833f84)

Actually you use reset.css